Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -720,6 +726,18 @@ public final class PackageBuilder {
         }
     }
 
+    /// Validates module alias key and value pairs and throws an error if empty, is a stdlib, or
+    /// contains invalid characters.
+    private func validateModuleAliases(_ aliases: [String: String]?) throws {
+        guard let aliases = aliases else { return }
+        for (entKey, entVal) in aliases {
```

Comment:
what does `entKey` `entVal` stand for? could we rename these variables more clearly?